packages/pg/package.js

```diff
@@ -67,11 +67,10 @@ Package.onUse(function(api) {
 });
 
 Package.onTest(function(api) {
-  api.use('tinytest');
-  api.use('simple:pg');
+  api.use(['tinytest','promise','simple:pg','ecmascript']);
+  api.imply('underscore');
```

Comment:
Ah. I should have removed that. I was having issues with client tests, one of which was _ being undefined. That was the how I fixed it. Unfortunately, there were other issues, so I removed client tests *pro tem* while I worked on them. I forget to take that out, though.